inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
+    sources: List[str] = field(default_factory=list)
     gpg_key_uri: Optional[str] = field(default=None)
     gpg_key_name: Optional[str] = field(default=None)
```

Comment:
**Suggestion:** For 'ApplicationAddSourceParameters', consider initializing 'gpg_key_uri' and 'gpg_key_name' with empty strings for consistency and to avoid None checks. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    class ApplicationAddSourceParameters:
        gpg_key_uri: str = field(default_factory=str)
        gpg_key_name: str = field(default_factory=str)
```